package.json

```diff
@@ -59,6 +59,7 @@
     "next-auth": "^4.24.6",
     "next-pwa": "^5.6.0",
     "next-seo": "^6.5.0",
+    "next-useragent": "^2.8.0",
```

Comment:
i dont think a separate package is needed
from memory, i think android devices can be checked using something like
```
const isAndroid = (): boolean => {
  return /Android/i.test(navigator.userAgent);
};
```

there is something similar for iOS too